jormungandr/src/settings/logging.rs

```diff
@@ -1,29 +1,29 @@
 use std::fmt::{self, Display};
 use std::fs;
-use std::io::{self, Write};
+use std::io::{self};
```

Comment:
Just `std::io`. Don't why the linters are not triggered.

Reply:
Done.